var/spack/repos/builtin/packages/libszip/package.py

```diff
@@ -36,11 +36,11 @@ class Libszip(AutotoolsPackage):
     homepage = "https://support.hdfgroup.org/doc_resource/SZIP/"
     url      = "https://support.hdfgroup.org/ftp/lib-external/szip/2.1.1/src/szip-2.1.1.tar.gz"
     list_url = "https://support.hdfgroup.org/ftp/lib-external/szip"
-    list_depth = 2
+    list_depth = 3
```

Comment:
There's really no point in even having a `list_url`. There are no other versions of `szip` available.

Reply:
There's version 2.1, which can't be downloaded without setting list_url=3.

Reply:
Ah, I see, they moved it to a `previous` directory. Didn't notice that the first time.